kelo_hitbot_driver/Axes pose
int32 toolID
int32 workpieceID
float32 speed
float32 acc
float32[] extAxesPos
